tests/unit/Command/FixEncryptedVersionTest.php

```diff
@@ -78,6 +78,7 @@ public static function setUpBeforeClass(): void {
 
 	public static function tearDownAfterClass(): void {
 		parent::tearDownAfterClass();
+		\OC_App::disable('encryption');
```

Comment:
Is this needed? I'm not sure if this will affect to other tests. We might want to run the whole test suite with encryption enable and this could disable encryption. This could mean that the rest of the suite will be run without encryption.